pages/UserScreen/UserScreen.jsx

```diff
@@ -0,0 +1,73 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { Text, View } from "react-native";
+import { useAppContext } from "../../shared/context/AppContext";
+import { styles } from "./UserScreen.styles";
+import { Icon, Divider, Button } from 'react-native-paper';
+
+export default function UserScreen({ navigation }) {
+  const user = useAppContext();
```

Comment:
Grâce à ça tu pourras faire `user.name` au lieu de `user.user.name`
```suggestion
  const { user, logout, deleteAccount } = useAppContext();
```